packages/fee-estimation/src/estimateFees.spec.ts

```diff
@@ -246,3 +249,15 @@ test('version should return the correct result', async () => {
   expect(await version(paramsWithRpcUrl)).toMatchInlineSnapshot('"1.0.0"')
   expect(await version(paramsWithViemClient)).toMatchInlineSnapshot('"1.0.0"')
 })
+
+test('basefeeScalar should return the correct result', async () => {
+  expect(await basefeeScalar({ ...paramsWithRpcUrl, ...functionDataBurn })).toMatchInlineSnapshot('1n')
+})
+
+test('blobBasefeeScalar should return the correct result', async () => {
+  expect(await blobBasefeeScalar({ ...paramsWithRpcUrl, ...functionDataBurn })).toMatchInlineSnapshot('1n')
+})
+
+test('blobBasefee should return the correct result', async () => {
+  expect(await blobBasefee({ ...paramsWithRpcUrl, ...functionDataBurn })).toMatchInlineSnapshot('0n')
```

Comment:
tests and typechecker for this package will take care of this being right. Only the sdk package is missing strong typing and tests